inject/src/main/java/io/avaje/inject/BeanScopeBuilder.java

```diff
@@ -216,16 +216,16 @@ default <D> BeanScopeBuilder provideDefault(Type type, Supplier<D> provider) {
   /**
    * Adds hooks that will execute after this scope is built.
    *
-   * @param runnables the PostConstruct hooks to add to the bean scope
+   * @param runnables the PostConstruct hooks to run after the BeanScope is constructed
    */
   BeanScopeBuilder addPostConstructHooks(Runnable... runnables);
 
   /**
-   * Adds hook that will execute after this scope is built.
+   * Adds hooks that will execute after this scope is built.
    *
-   * @param consumer the PostConstruct hook to add to the bean scope
+   * @param consumers the PostConstruct hooks to run after the BeanScope is constructed
    */
-  BeanScopeBuilder addPostConstructConsumerHook(Consumer<BeanScope> consumer);
+  BeanScopeBuilder addPostConstructHooks(BeanScopeConsumer... consumers);
```

Comment:
I suspect this is only ever called with 1 consumer.  I suspect the same for the Runnable hooks.

... so I _think_ these should both be changed to not use var-args ?

Reply:
Oh yeah, we could also do that